### PR TITLE
ios applink  on the landing page and bottom notifciation

### PR DESCRIPTION
**Pull request by owocki**
##### Description

ios applink  on the landing page and bottom notifciation

screenshot: http://bits.owocki.com/292k1c3n0c1G/Screen%20Shot%202018-05-07%20at%209.15.34%20AM.png

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

ui

##### Refers/Fixes

self

### CODE REVIEW

File: app/retail/templates/index.html [hunk 1]
```diff
@@ -417,6 +417,35 @@ <h4 class="">{% trans "Press" %}</h4>
...
+        <div class="row pb-1">
+          <div class="col text-center">
+            <h4 class="">{% trans "iOS App Now Available" %}</h4>
+          </div>
+        </div>
+        <div class="row pb-1">
+          <div class="col d-flex flex-column align-items-center">
+            <div class="explain-img-wrapper mb-3 d-flex justify-content-center align-items-center" style="min-height: 200px;">
+            <a href="{% url "ios" %}">
+                <img style="min-height: 200px;" src="{% static "v2/images/ios/iphone_crop.png" %}" >
```
Comment (thelostone-mc) [inline app/retail/templates/index.html]: replace it with `max-height: 1rem` ? cause we are shifting to `rem` + that way this doesn't seem to big on mobile devices
we should move this to css file too!

File: app/retail/templates/index.html [hunk 2]
```diff
@@ -417,6 +417,35 @@ <h4 class="">{% trans "Press" %}</h4>
...
+            </a>
+            </div>
+          </div>
+          <div class="col d-flex flex-column align-items-center">
+            <div class="justify-content-left align-items-left pt-3" style="text-align: left;">
```
Comment (thelostone-mc) [inline app/retail/templates/index.html]: `align-items-left` and `text-align: left;` ? Do we need both

File: app/retail/templates/index.html [hunk 3]
```diff
@@ -417,6 +417,35 @@ <h4 class="">{% trans "Press" %}</h4>
...
+                Be the first to grab the best funded OSS work out there, on the go.
+              </p>
+              <a class="btn btn-lg btn-warning" role="button" href="{% url "ios" %}">{% trans "Get The App" %}</a>
+            </div>
+          </div>
+        </div>
+        <div class="row pb-5">
+          <div class="col d-flex flex-column align-items-center">
+            </div>
+        </div>
```
Comment (thelostone-mc) [inline app/retail/templates/index.html]: This seems redundant , could we remove this ?

File: app/retail/templates/index.html [hunk 4]
```diff
@@ -417,6 +417,35 @@ <h4 class="">{% trans "Press" %}</h4>
           </div>
         </div>
       </div>
+      <div class="container-fluid pt-0" style="background-color: #eee;">
```
Comment (thelostone-mc) [inline app/retail/templates/index.html]: the `background-color` seems to be overridden [here](https://d3vv6lp55qjaqc.cloudfront.net/items/1f0v0R0k1A3T1q471q0Q/Screen%20Shot%202018-05-07%20at%209.15.34%20AM.png)

**Comment by thelostone-mc** [comment]
@owocki @mbeacom left a few comments! If it seems like a log -> we could get this merged and i'll throw in an update ^_^

**Comment by thelostone-mc** [comment]
@PixelantDesign should we include this in the redesign ?

**Comment by thelostone-mc** [comment]
Since this page is due for revamp  -> I could hardcode the css and make it looks like this
Your call @owocki / @mbeacom ^_^


screen shot 2018-05-08 at 4 44 19 pm

**Comment by owocki** [comment]
@thelostone-mc sure -- pls just update my PR !

**Comment by thelostone-mc** [comment]
@mbeacom UI skill level +1

x

**Comment by owocki** [comment]
shit... this was not supposed to go out until @PixelantDesign signed off on the iOS app. we are still devving it

**Comment by owocki** [comment]
5ebf6b48a9add29b951654af8af7d1e246764aa0

**Comment by owocki** [comment]
https://github.com/gitcoinco/web/pull/1167

**Comment by mbeacom** [comment]
👍

**Pull request merged**